client/plots/test/oncomatrix.spec.js

```diff
@@ -305,11 +297,56 @@ tape('2 genes, 2 dict terms, divideBy', function (test) {
 	}
 })
 
-tape('launch matrix using runproteinpaint with launchGdcMatrix', function (test) {
-	test.timeoutAfter(100000)
+tape('2 geneVariant, 2 dict terms, 1 gene exp, 1 survival', function (test) {
+	test.timeoutAfter(20000)
 	test.plan(1)
+
+	const termlst2 = [...termlst, { term: { name: 'MYC', type: 'geneExpression' } }, { id: 'Overall Survival' }]
+	runpp({
+		state: {
+			nav: { header_mode: 'hidden' }, // must set to hidden for gdc, since it lacks termdb method to get cohort size..
+			plots: [
+				{
+					chartType: 'matrix',
+					settings: {
+						matrix: {
+							// the matrix autocomputes the colw based on available screen width,
+							// need to set an exact screen width for consistent tests using getBBox()
+							availContentWidth: 1200
+						}
+					},
+					termgroups: [{ lst: termlst2 }]
+				}
+			]
+		},
+		matrix: {
+			callbacks: {
+				'postRender.test': runTests
+			}
+		}
+	})
+
+	async function runTests(matrix) {
+		matrix.on('postRender.test', null)
+
+		const numSeries = termlst2.length + 2 // why must plus 2?
```

Comment:
it seems like by including geneExp and survival terms, the `sjpp-matrix-term-label-g` series count must increment by 2. @congyu-lu do you know why?

Reply:
caused by the scale for MYC gene expression

Reply:
just fixed it